src/cljs/chronograph_web/http.cljs

```diff
@@ -17,3 +17,10 @@
           :format          (ajax/json-request-format)
           :response-format (ajax/json-response-format {:keywords? true})}
          http-xhrio-map))
+
+(defn put [uri http-xhrio-map]
+  (merge {:method :put
+          :uri uri
+          :format          (ajax/json-request-format)
+          :response-format (ajax/json-response-format {:keywords? true})}
+         http-xhrio-map))
```

Comment:
If someone sends `:method :post` in the `http-xhrio-map`, it will override the :put.

I think the correct thing would be to swap the maps, and merge our map into the `http-xhrio-map`.

Reply:
Marking this resolved as discussed on the call.